DEBUG 2017-08-14 05:15:54,647 /root/enjoylove/lib/python2.7/site-packages/django/template/base.py base.py base _resolve_lookup 925: Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/root/enjoylove/lib/python2.7/site-packages/django/template/base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [name] in u"<RegexURLResolver <module 'grappelli.urls' from '/root/enjoylove/lib/python2.7/site-packages/grappelli/urls.pyc'> (None:None) ^grappelli/>"
DEBUG 2017-08-14 05:15:54,648 /root/enjoylove/lib/python2.7/site-packages/django/template/base.py base.py base _resolve_lookup 925: Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/root/enjoylove/lib/python2.7/site-packages/django/template/base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [name] in u'<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>'
DEBUG 2017-08-14 05:15:54,649 /root/enjoylove/lib/python2.7/site-packages/django/template/base.py base.py base _resolve_lookup 925: Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/root/enjoylove/lib/python2.7/site-packages/django/template/base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [name] in u"<RegexURLResolver <module 'rest_framework.urls' from '/root/enjoylove/lib/python2.7/site-packages/rest_framework/urls.pyc'> (rest_framework:rest_framework) ^api-auth/>"
DEBUG 2017-08-14 05:15:54,649 /root/enjoylove/lib/python2.7/site-packages/django/template/base.py base.py base _resolve_lookup 925: Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/root/enjoylove/lib/python2.7/site-packages/django/template/base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [name] in u"<RegexURLResolver <module 'enjoy_love.urls' from '/root/enjoylove/enjoylove/enjoy_love/urls.pyc'> (None:None) ^user/>"
DEBUG 2017-08-14 05:15:54,650 /root/enjoylove/lib/python2.7/site-packages/django/template/base.py base.py base _resolve_lookup 925: Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/root/enjoylove/lib/python2.7/site-packages/django/template/base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [name] in u"<RegexURLResolver <module 'rest_framework_docs.urls' from '/root/enjoylove/lib/python2.7/site-packages/rest_framework_docs/urls.pyc'> (None:None) ^docs/>"
